src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -23,6 +23,7 @@ public EndpointParameter(Endpoint endpoint, IParameterSymbol parameter, WellKnow
     {
         Ordinal = parameter.Ordinal;
         IsOptional = parameter.IsOptional();
+        HasDefaultValue = parameter.HasExplicitDefaultValue;
```

Comment:
We add tracking for whether or not a method has a default parameter so that we can pass it in the concrete delegate type to support [default parameter values in lambdas](https://github.com/dotnet/csharplang/blob/main/proposals/lambda-method-group-defaults.md#optional-and-parameter-array-parameters-for-lambdas-and-method-groups).